src/api/list_add.sql

```diff
@@ -0,0 +1,60 @@
+DROP FUNCTION IF EXISTS api.list_add(jsonb);
+
+CREATE OR REPLACE FUNCTION api.list_add(data jsonb)
+RETURNS void AS $$
+DECLARE  
+	class          jsonb;
+	obj_id 		   uuid;
+	value_to_add   jsonb; 
+	json_path      text[];
+	obj		 	   jsonb;
+	new_obj		   jsonb;
+	field_value    jsonb;
+	access_token   jsonb;
+
+	BEGIN
+		class := data->'class';
+		IF (class IS NULL) THEN
+			RAISE EXCEPTION 'The reclada object class is not specified';
+   		END IF;
+	
+   	
+		obj_id := (data->>'id')::uuid;
+		IF(obj_id IS NULL) THEN
+        	RAISE EXCEPTION 'The is no id';
+    	END IF;
+	
+		access_token := data->'access_token';
+		SELECT api.reclada_object_list(format(
+			'{"class": %s, "attrs": {}, "id": "%s", "access_token": %s}',
+        	class,
+        	obj_id,
+        	access_token
+    		)::jsonb) -> 0 INTO obj;
+    	
+    	IF (obj IS NULL) THEN
+        	RAISE EXCEPTION 'The is no object with such id';
+    	END IF;
+    	
+    	json_path := format('{attrs, %s}', data->>'field');
```

Comment:
Why should json path be prepared here and used for acquiring the field value, isn't the field itself (obj->>field) sufficient?

Reply:
Didn't change it because I use json_path several times during the function